src/scripts/init.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS public.flex_versions
```

Comment:
Where are these scripts used?

Reply:
This is destination table where we persist the information in DB.